src/main/java/racingcar/domain/Racing.java

```diff
@@ -1,28 +1,33 @@
-package racingcar;
+package racingcar.domain;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Random;
 
 public class Racing {
 
-    private static final Random RANDOM = new Random();
+    public static final int MAX_BOUND = 10;
 
     private final List<Car> cars;
```

Comment:
`Cars`에서 값을 꺼내서 `List<Car>`를 전달하는 대신에, `Cars`를 그대로 사용해보면 어떨까요?
```suggestion
    private final Cars cars;
```